test/framework/easyconfigs/yeb/goolf-1.4.10.yeb

```diff
@@ -0,0 +1,38 @@
+easyblock: Toolchain
+
+name: goolf
+version: 1.4.10
+
+homepage: (none)
+description: |
+    GNU Compiler Collection (GCC) based compiler toolchain, including
+     OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK.
+
+toolchain: {name: dummy, version: dummy}
+
+vars:
```

Comment:
defining these variables should be done on top, before actually defining easyconfig parameters?

maybe we should use something like `_internal_variables_` as 'key' here (just as a style thing), to make it clear?